src/webview/webview2/mod.rs

```diff
@@ -74,7 +81,12 @@ impl InnerWebView {
     let file_drop_handler = attributes.file_drop_handler.take();
     let file_drop_window = window.clone();
 
-    let env = Self::create_environment(&web_context, pl_attrs.clone(), attributes.autoplay)?;
+    let env = Self::create_environment(
+      &web_context,
+      pl_attrs.clone(),
+      attributes.autoplay,
+      attributes.proxy_config.clone(),
```

Comment:
Could probably just pass a reference instead of cloning here

Reply:
Actually I think we might still need a clone here, same as the reason why `pl_attrs` is cloned here originally.

Inside `create_environment`, `CreateCoreWebView2EnvironmentCompletedHandler::wait_for_async_operation` will cause a lifetime problem.

Reply:
right, maybe we should then construct the browser args string before calling `CreateCoreWebView2EnvironmentCompletedHandler::wait_for_async_operation`, that way we don't have lifetime issues and do one string allocation instead of two. Same could probably be done for `pl_atts` if possible.